### ISSUE TITLE
learning rate and warmup settings for 32 GPUs trainning

### ISSUE THREAD

**Issue by Wanglm2TC** [ultralytics/yolov5#12412]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello, I have been using YOLOv5 to reproduce official accuracy recently.There is a scenario where 32 GPUs are used for training, so I have some questions about parameter settings:

1. First of all, it needs to be confirmed that when reproducing the official results on coco, except for yolov5n and yolov5s using hyp.scratch-low.yaml, all other models use the hyp.scratch-high.yaml?
2. I have searched YOLOv5 issues and recommend. It is recommended to use the default settings about learning rate for any batch size and multi card. How about 32 GPUs trainning? And I noticed that the loss is scaled by batchsize, so the learning rate can be maintained unchanged. Is warmup the same?

Best regards,

**Comment by glenn-jocher** [comment]
@Wanglm2TC hi there! Thanks for reaching out. Yes, for reproducing the official results on COCO, except for YOLOv5n and YOLOv5s using hyp.scratch-low.yaml, all other models use the hyp.scratch-high.yaml.

Regarding the learning rate and warm-up for 32 GPUs training, the recommendation is to use the default settings for learning rate and warm-up for any batch size and multi-card setup. As you mentioned, the loss is scaled by batch size, so the learning rate can remain unchanged and warm-up can follow the same principle.

Best regards!
